insonmnia/auth/auth.go

```diff
@@ -24,9 +24,9 @@ func (e Event) String() string {
 // By default the router allows unregistered events, but this behavior can be
 // changed using `DenyUnregistered` option.
 type AuthRouter struct {
+	mu        sync.RWMutex
 	ctx       context.Context
 	log       *zap.Logger
-	mu        sync.RWMutex
```

Comment:
Leave out of the scope changes for another PR. Changing all the things everywhere can cause conflicts.